dev/tests/integration/testsuite/Magento/Framework/DB/Adapter/Pdo/MysqlTest.php

```diff
@@ -34,6 +34,37 @@ protected function tearDown(): void
         restore_error_handler();
     }
 
+    /**
+     * Check PDO stringify fetches options
+     */
+    public function testStringifyFetches(): void
+    {
+        $tableName = $this->resourceConnection->getTableName('table_with_int_column');
+        $columnId = 'integer_column';
+        $adapter = $this->getDbAdapter();
+
+        $table = $adapter
+            ->newTable($tableName)
+            ->addColumn($columnId, Table::TYPE_INTEGER);
+        $adapter->createTable($table);
+        $adapter->insert($tableName, [$columnId => 100]);
+
+        $select = $adapter->select()
+            ->from($tableName)
+            ->columns([$columnId])
+            ->limit(1);
+        $result = $adapter->fetchOne($select);
+
+        $isStringify = $adapter->getConfig()['driver_options'][\PDO::ATTR_STRINGIFY_FETCHES] ?? null;
+        if ($isStringify || $isStringify === null) {
```

Comment:
few notes here:
- it is generally bad practice to have checks wrapped in conditional statement. It is good sign that you need to have two tests or use data provider
- Am I right that `PDO::ATTR_STRINGIFY_FETCHES` always will be true in this test ?(I don't see that we update config in test).

Reply:
@xmav  It will always be true, but I think we'll have a possibility to pass 'false' when we add --db-stringify-fetches option for bin/install.
I can remove that condition if we're not going to add the option in the future.

Reply:
yes please. remove it from this test. Config could be passed without extra method thought `__construct`
I would suggest to have test to verify that value set in code could be overwritten from config. May be `dev/tests/integration/testsuite/Magento/Framework/Model/ResourceModel/Type/Db/Pdo/MysqlTest.php` will be good place for this test

Reply:
Hello @xmav 
Fixed, please check.
Thank you in advance